packages/components/select/src/select.stories.ts

```diff
@@ -16,42 +19,149 @@ const onSubmit = (event: Event & { target: HTMLFormElement }): void => {
   },
-  render: ({ maxOverlayHeight }) => html`
+  render: ({ maxOverlayHeight, size }) => html`
+    <style>
+      sl-select {
+        width: 400px;
+        max-width: 90vw;
+        display: inline-flex;
+      }
+    </style>
     <sl-button>To focus</sl-button>
-    <sl-select maxOverlayHeight=${maxOverlayHeight}>
+    <sl-select maxOverlayHeight=${maxOverlayHeight} .size="${size}">
       <sl-select-option-group group-title="Happy">
         <sl-select-option>😄 Grinning Face with Smiling Eyes</sl-select-option>
-        <sl-select-option selected>😂 Face with Tears of Joy</sl-select-option>
+        <sl-select-option>😂 Face with Tears of Joy</sl-select-option>
         <sl-select-option>😊 Smiling Face with Smiling Eyes</sl-select-option>
         <sl-select-option>🤩 Star-Struck</sl-select-option>
         <sl-select-option disabled>🙂 Slightly Smiling Face</sl-select-option>
         <sl-select-option>🥳 Partying Face</sl-select-option>
       </sl-select-option-group>
+      <sl-select-option>😶 Unfazed</sl-select-option>
       <sl-select-option-group group-title="Sad">
         <sl-select-option>😒 Unamused Face</sl-select-option>
         <sl-select-option>🤧 Sneezing Face</sl-select-option>
         <sl-select-option>😓 Downcast Face with Sweat</sl-select-option>
         <sl-select-option>😡 Enraged Face</sl-select-option>
       </sl-select-option-group>
+      <sl-select-option-group>
+        <sl-select-option>🐷 Pig</sl-select-option>
+        <sl-select-option selected>🐨 Koala</sl-select-option>
+        <sl-select-option>🐼 Panda</sl-select-option>
+        <sl-select-option>🦊 Fox</sl-select-option>
+      </sl-select-option-group>
       <sl-select-option>🤖 Robot</sl-select-option>
     </sl-select>
     <sl-button>To focus</sl-button>
   `
 };
 
+const options = html`
+  <sl-select-option>🐷 Pig</sl-select-option>
+  <sl-select-option selected>🐨 Koala</sl-select-option>
+  <sl-select-option>🐼 Panda</sl-select-option>
+  <sl-select-option>🦊 Fox</sl-select-option>
+`;
+export const All: StoryObj = {
+  render: () => {
+    return html` <style>
+        table {
+          border-collapse: collapse;
+        }
+
+        th {
+          text-transform: capitalize;
+        }
+        th,
+        td {
+          padding: 4px 8px;
+        }
+        thead td {
+          text-align: center;
+        }
+      </style>
+      ${sizes.map(
+        size => html`<h2>Size: ${size}</h2>
+          <table>
+            <thead>
+              <tr>
+                <td></td>
+                ${disabledStates.map(
+                  disabledState =>
+                    html` <td class="${disabledState ? 'sb-disabled' : ''}">
+                      ${disabledState ? 'Disabled' : 'Enabled'}
+                    </td>`
+                )}
+              </tr>
+            </thead>
+            <tbody>
+              ${states.map(
+                state => html` <tr>
+                  <th>${state}</th>
+                  ${disabledStates.map(
```

Comment:
Selected option disabled has a blue or grey border on the left?
In the story we have a version with blue:
![image](https://github.com/sl-design-system/components/assets/111562742/c57a656a-68b7-4bc4-9b5d-e0055ab8a9f0)


But in Figma with grey?
![image](https://github.com/sl-design-system/components/assets/111562742/2adb4235-5e6c-4de0-b1c3-2a95ce989900)